src/textual/css/parse.py

```diff
@@ -305,7 +306,7 @@ def substitute_references(
 
 
 def parse(
-    css: str, path: str, variables: dict[str, str] | None = None
+    css: str, path: str | PurePath, variables: dict[str, str] | None = None
```

Comment:
nothing else to do here, as we only pass this `path` to the TokenizerState class